web/src/components/tooltips/TooltipWrapper.tsx

```diff
@@ -19,18 +19,41 @@ export default function TooltipWrapper({
   children,
   side,
   sideOffset,
-  tooltipClassName,
+  tooltipId,
 }: TooltipWrapperProperties): ReactElement {
-  const [isOpen, setIsOpen] = useState(false);
+  const [openTooltipId, setOpenTooltipId] = useAtom(openTooltipIdAtom);
+  const [localIsOpen, setLocalIsOpen] = useState(false);
   const isMobile = useIsMobile();
   if (!tooltipContent) {
     return children;
   }
 
   // Helpers
-  const openTooltip = () => setIsOpen(true);
-  const closeTooltip = () => setIsOpen(false);
-  const toggleTooltip = () => setIsOpen(!isOpen);
+  const openTooltip = () => {
```

Comment:
All of these need to be in useCallback hooks to prevent re-renders. Since you are changing it anyway do you mind adding that?

Reply:
Gotcha, will do